ui/components/component-library/checkbox/README.mdx

```diff
@@ -41,7 +41,7 @@ Use the `isChecked` boolean prop to set the checked state of the `Checkbox`
 ```jsx
 import { Checkbox } from '../../component-library';
 
-<Checkbox isChecked={true} />;
+<Checkbox isChecked={true} label="isChecked demo" />;
```

Comment:
Matches code example to story a little more

<img width="461" alt="Screenshot 2023-07-13 at 2 06 38 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/611e1360-1c4e-4a8f-a50c-a4ee70e17624">